### ISSUE TITLE
el-submenu do not expand correctly when change route

### ISSUE THREAD

**Issue by xyorz** [element-plus/element-plus#1006]
I tried to use el-menu with router option, but it behaves strange.
When route change, the submenu doesn't expend correctly if the menu item extracted into a component.
But it works well if the menu item directly wrapped by el-menu.
Here is the demo:
[https://codesandbox.io/s/sad-stallman-dl3n0?file=/src/NavbarMenu.vue](https://codesandbox.io/s/sad-stallman-dl3n0?file=/src/NavbarMenu.vue)

**Comment by Liiked** [comment]
Thx for reporting this issue 😉, we'll fix it ASAP.

**Comment by JeremyWuuuuu** [comment]
So the first investigation I made, I found facts:

1. The route does not expand correctly
2. The bug only impact route `3-1` and `5-1` for `2-1` `4-1` `6-1` it expands correctly.

Check this out:


https://user-images.githubusercontent.com/15975785/103764466-7aa56000-5056-11eb-9047-6e84cc7cfdc5.mov

So I do have a theory that the route only expands for even numbers, I am going to find out the correctness of my assumption.
